bycycle/tests/test_plts_cyclepoints.py

```diff
@@ -0,0 +1,16 @@
+"""Tests plotting cyclepoints."""
+
+import pytest
+
+from bycycle.plts.cyclepoints import plot_cyclepoints
+from bycycle.tests.utils import plot_test
+
+###################################################################################################
+###################################################################################################
+
+@plot_test
+@pytest.mark.parametrize("tlims", [None, (0, 1)])
+def test_plot_cyclepoints(tlims, sim_args):
+    """Test plotting extrema/zero-crossings."""
```

Comment:
For consistently, we generally don't add docstrings for test funcs, so since this one doesn't say anything special, I think we could just drop it.